proposals/accepted/051-ghc-base-libraries.rst

```diff
@@ -0,0 +1,330 @@
+================
+GHC's base libraries
+================
+----------
+Combining stability with innovation
+----------
+
+:Date: June 2023
+:Authors:
+  John Ericson — secretary,
+  Ben Gamari — GHC,
+  Adam Gundry — GHC,
+  Andrew Lelechenko — CLC,
+  Julian Ospald — CLC,
+  Simon Peyton Jones — GHC
+
+.. sectnum::
+.. contents::
+
+Introduction
+=========
+
+This document describes an plan agreed between the GHC Team and the Core Libraries Committee, saying how we plan to work in partnership to reconcile the goals of innovation and stability for GHC and its ecosystem.
+
+Formally, then, it is not so much a proposal as a record of an outcome.
+We are, nevertheless, using the Haskell Foundation Technical Working Group proposals repo, so that we can have a permanent public record of how we plan to work, so that others can comment, and so that the document can be polished to add clarity where necessary.
+
+Goals
+=====
+
+This proposal seeks to reconcile two goals, both of which are complex and multi-faceted.
+
+1. **The Core Libraries Committee seeks to exercise its control over decisions affecting 'base' (API, performance, semantics), including its dependencies.**
+   Managing API changes of ``base`` is the CLC's primary mandate.
+   If ``base`` changes gratuitously, everyone suffers.
+
+2. **The GHC team seeks the freedom to:**
+
+   - Innovate in the language design.
+     GHC has hundreds of extensions, and people suggest more all the time, via the GHC Proposals process.
+
+   - Move rapidly to fix bugs, improve performance, and refactor GHC's internals to pay down technical debt.
+
+The two goals are in risk of conflict, and we have seen such conflict in the past.
+The purpose of this proposal is to reconcile these goals, in a way that satisfies all parties.
+
+Each of us places different emphasis on these goals, but respects them both.
+The CLC does not want to constrain GHC unnecessarily; and the GHC team does not want to undermine CLC's efforts, indeed quite the reverse!
+
+The proposal is based on `HF Proposal: standard library reform <https://github.com/haskellfoundation/tech-proposals/pull/47>`__, but is independent of it.
```

Comment:
Add: 

You can find more background in [CLC issue #146](https://github.com/haskell/core-libraries-committee/issues/146).

Reply:
Like this?
```suggestion
The proposal is based on `HF Proposal: standard library reform <https://github.com/haskellfoundation/tech-proposals/pull/47>`__, but is independent of it.
You can find more background in `CLC issue #146 <https://github.com/haskell/core-libraries-committee/issues/146>`__.
```

Reply:
Yes, that's what I was thinking.  Just to keep pointers to history.